src/Stores/SubjectsPromises/actions.js

```diff
@@ -0,0 +1,31 @@
+/* @flow */
+import type { Dispatch, SubjectPromise } from '../../types';
+import { getSubjectsPromises as apiGetSubjectsPromises } from '../../utils/api';
+
+const getSubjectsPromises = () => ({ type: 'SUBJECTS_PROMISES_LOAD' });
```

Comment:
Abstract the types into constants and use those both here and in the reducer. Make it less likely there is a spelling error and such

Reply:
Will do.
Flow helps with the spelling mistakes here though.

If I remember correctly Flow gets confused if we use variables here...I might be misremembering though

Reply:
any thoughts on which file these types should live?